staging/src/k8s.io/apiserver/pkg/apis/flowcontrol/bootstrap/default.go

```diff
@@ -82,6 +82,12 @@ var (
 	}
 )
 
+func makeInt32Ptr(v int32) *int32 {
```

Comment:
nit: use `k8s.io/utils/pointer#Int32()` instead?

Reply:
i am using the util function now